core/src/main/java/org/apache/accumulo/core/client/admin/TabletInformation.java

```diff
@@ -59,7 +59,10 @@ public interface TabletInformation {
   String getTabletState();
 
   /**
-   * @return the Location of the tablet as a String.
+   * @return the Location of the tablet as a String or empty if the location in the TabletMetadata
+   *         does not exist. When not empty, the String will be of the form
+   *         "{@code <location type>:<host>:<port>}", where the location type is one of
+   *         {@code LocationType}.
```

Comment:
LocationType enum is defined outside of the public API, also this will only return two of the enums three values.  Could just list them in the javadoc.

```suggestion
   *        {@code CURRENT} or {@code FUTURE}
```